hikari/impl/rest.py

```diff
@@ -1253,15 +1259,15 @@ async def _create_message(
 
         if undefined.all_undefined(embed, embeds) and isinstance(content, embeds_.Embed):
             # Syntactic sugar, common mistake to accidentally send an embed
-            # as the content, so lets detect this and fix it for the user.
+            # as the content, so let's detect this and fix it for the user.
             embed = content
             content = undefined.UNDEFINED
 
         elif undefined.all_undefined(attachment, attachments) and isinstance(
```

Comment:
What about if attachment or attachments is None

Reply:
We dont allow attachments to be None atm, but this is syntactic sugar for passing it to the content. If they want to pass any of those as None, they can just pass it to the argument

Reply:
Sry i put this on the wrong resource